frontend/app-development/hooks/queries/useNewsListQuery.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import { useServicesContext } from 'app-shared/contexts/ServicesContext';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import { NewsList } from 'app-shared/types/api/NewsList';
+
+export const useNewsListQuery = (): UseQueryResult<NewsList> => {
+  const { getNewsList } = useServicesContext();
+  return useQuery<NewsList>([QueryKey.NewsList], () => getNewsList('nb'), {
+    cacheTime: Infinity,
+    staleTime: Infinity,
```

Comment:
Is there any reason to add this config directly here? 😊

Reply:
Only to ensure that we do not re-fetch this data. Is there some global config that sets this for all queries?